.github/workflows/scorecards.yml

```diff
@@ -27,7 +27,7 @@ jobs:
           persist-credentials: false
 
       - name: "Run analysis"
-        uses: ossf/scorecard-action@865b4092859256271290c77adbd10a43f4779972 # tag=v2.0.3
+        uses: ossf/scorecard-action@0864cf19026789058feabb7e87baa5f140aac736 # tag=v2.3.1
```

Comment:
Should we follow this format for other actions also? or vice versa. It has been discussed once in https://github.com/containerd/containerd/pull/7404. But want to recheck if we should be consistent across all the workflow files.

Reply:
👍

Reply:
We will update in a separate PR to using tags if needed.

Reply:
Do we know if `ossf` is configurable to have different rules for "trusted" and "less trusted" actions?

My concern with pinning at times is that it also means

- they must always be explicitly updated (i.e. release branches will likely use obsolete versions, and don't get minor/patch updates)
- is there a verification that checks if `<digest>` matches `tag`? Can it automatically resolve? (there's no easy way to verify that `0864cf19026789058feabb7e87baa5f140aac736` is _ACTUALLY_ `v2.3.1`; this makes it actually a _higher_ risk of using a wrong version.